yang/frr-zebra.yang

```diff
@@ -1985,4 +1989,63 @@ module frr-zebra {
   }
 
   // End interface model augmentation
+
+  augment "/frr-route-map:lib"
+    + "/frr-route-map:route-map"
+    + "/frr-route-map:entry"
+    + "/frr-route-map:match-condition"
+    + "/frr-route-map:condition-value" {
+    case ipv4-prefix-length {
+      when "./condition = 'ipv4-prefix-length' or
+            ./condition = 'ipv4-next-hop-prefix-length'";
+      leaf ipv4-prefix-length {
+        type uint8 {
+          range "0..32";
+        }
+      }
+    }
+    case ipv6-prefix-length {
+      when "./condition = 'ipv6-prefix-length'";
+      leaf ipv6-prefix-length {
+        type uint8 {
+          range "0..128";
+        }
+      }
+    }
```

Comment:
This set option for zebra seems to be missing.

match ipv6 next-hop type 
  blackhole  Blackhole

Reply:
It is already implemented here: https://github.com/FRRouting/frr/blob/master/lib/routemap_cli.c#L505